plugins/gimbal/gimbal.h

```diff
@@ -84,6 +84,40 @@ class Gimbal : public PluginBase
      */
     void set_pitch_and_yaw_async(float pitch_deg, float yaw_deg, result_callback_t callback);
 
+
+    /**
+     * @brief Set gimbal region of interest (ROI).
+     *
+     * This sets a region of interest that the gimbal will point to.
+     * The gimbal will continue to point to the specified region until it
+     * receives a new command.
+     * The function will return when the command is accepted, however, it might
+     * take the gimbal longer to actually rotate to the ROI.
+     *
+     * @param latitude_deg Latitude in degrees.
+     * @param longitude_deg Longitude in degrees.
+     * @param altitude_m Altitude in meters (ASML).
```

Comment:
Is this relative or absolute? Nevermind, it's ASML. It's usually refered to as absolute altitude for DroneCore.